config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Refinery::Core::Engine.routes.draw do
   namespace :blog, :path => Refinery::Blog.page_url do
     root :to => "posts#index"
-    resources :posts, :only => [:show]
+    resources :posts, path: '', :only => [:show] do
```

Comment:
I'm ok if we want to use blog urls without `posts` level but i would also like to add a redirect to be backward  compatible with old urls